eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/NodeRelay.scala

```diff
@@ -259,7 +260,7 @@ class NodeRelay private(nodeParams: NodeParams,
 
   private def doSend(upstream: Upstream.Hot.Trampoline, nextPayload: IntermediatePayload.NodeRelay, nextPacket_opt: Option[OnionRoutingPacket]): Behavior[Command] = {
     context.log.debug(s"relaying trampoline payment (amountIn=${upstream.amountIn} expiryIn=${upstream.expiryIn} amountOut=${nextPayload.amountToForward} expiryOut=${nextPayload.outgoingCltv})")
-    relay(upstream, nextPayload, nextPacket_opt)
+    relay(upstream, nextPayload, nextPacket_opt, confidence = 0.5)
```

Comment:
Same here, we should relay the incoming endorsement by default. What should be the behavior for trampoline? Should use the `min` of the incoming values?

Reply:
We now copy the input value, using the minimum in case we've aggregated incoming HTLCs.